kai/reactive_codeplanner/main.py

```diff
@@ -106,11 +107,23 @@ def main() -> None:
     init_logging_from_config(kai_config)
     model_provider = ModelProvider(kai_config.models)
 
+    analyzer = AnalyzerLSP(
+        analyzer_lsp_server_binary=Path(args.analyzer_lsp_server_binary),
+        repo_directory=Path(args.source_directory),
+        rules_directory=Path(args.rules_directory),
+        analyzer_lsp_path=Path(args.analyzer_lsp_path),
+        analyzer_java_bundle_path=Path(args.analyzer_lsp_java_bundle),
+        dep_open_source_labels_path=Path(),
```

Comment:
add `dep_open_source_labels_path` to args and default to None, I saw this causing problems